src/main/scala/com/codacy/client/bitbucket/v2/AccessToken.scala

```diff
@@ -1,15 +1,21 @@
 package com.codacy.client.bitbucket.v2
 
 import play.api.libs.functional.syntax._
-import play.api.libs.json.{__, Reads}
+import play.api.libs.json.{__, Format, Json, Reads}
 
-case class AccessToken(access_token: String, refresh_token: String)
+case class AccessToken(access_token: String, scopes: String, expires_in: Int, refresh_token: String, token_type: String)
```

Comment:
Please leva the refresh_token: String as second argument, let's avoid shuffle variables around of the same type without a very good reason